galaxy.yml

```diff
@@ -1,6 +1,6 @@
 namespace: cisco
 name: catalystwan
-version: 0.2.2
+version: 0.2.3
```

Comment:
It should go as 0.3.0, cluster changes are not patch only ;)

Reply:
done